docs/guides/platform/billing-and-support/billing-and-payments/index.md

```diff
@@ -22,31 +22,31 @@ tags: ["linode platform"]
 
 ## How Hourly Billing Works
 
-All services are billed automatically at the end of the month. If you used a service for the entirety of the past month, you'll be billed the **monthly cap** amount for that service. If you used a service for only part of the past month, you'll be billed at the **hourly** rate for that service. If your usage during any given month hits the monthly cap for the service, hourly billing stops. You'll never be billed more than the monthly cap for any service, excluding network [transfer overages](/docs/platform/billing-and-support/network-transfer-quota/#how-overages-work). In other words, if you've used a service for the entire month, you'll have a predictable amount on your bill.
+All services are billed automatically at the end of the month. If you used a service for the entirety of the past month, you'll be billed the **monthly cap** amount for that service. If you used a service for only part of the past month, you'll be billed at the **hourly** rate for that service. If your usage during any given month hits the monthly cap for the service, hourly billing stops. You'll never be billed more than the monthly cap for any service, excluding network [transfer overages](/docs/guides/network-transfer/#how-overages-work). In other words, if you've used a service for the entire month, you'll have a predictable amount on your bill.
 
 ## Linode Cloud Hosting and Backups
 
 Full specs of each plan can be found on our [pricing page](https://www.linode.com/pricing)
 
-A large base amount of network transfer is included with all Linode plans, but exceeding that amount can result in a [transfer overage](/docs/platform/billing-and-support/network-transfer-quota/#how-overages-work) charge beyond the monthly cap.
+A large base amount of network transfer is included with all Linode plans, but exceeding that amount can result in a [transfer overage](/docs/guides/network-transfer/#how-overages-work) charge beyond the monthly cap.
 
-Due to the [impending exhaustion of the IPv4 address space](http://en.wikipedia.org/wiki/IPv4_address_exhaustion), Linode requires users to provide technical justification in order to add extra public IPv4 addresses to an instance. If you need access to additional IPv4 addresses, please [contact Support](/docs/platform/billing-and-support/support/) with your justification.
+Due to the [impending exhaustion of the IPv4 address space](http://en.wikipedia.org/wiki/IPv4_address_exhaustion), Linode requires users to provide technical justification in order to add extra public IPv4 addresses to an instance. If you need access to additional IPv4 addresses, please [contact Support](/docs/guides/support/) with your justification.
 
 ## Object Storage
 
-Review our [Object Storage Pricing and Limitations](/docs/platform/object-storage/pricing-and-limitations/) guide for information on Object Storage pricing and how Object Storage interacts with your account's network transfer pool.
```

Comment:
I struggled with balancing how much we should change the wording of the links and if we should validate that each new link is still accurate based on the context. At this point, I'm not sure we need to think too critically about it. The link already exists, but just redirects to another page. This script just gets rid of that redirect. Like you mentioned, we could always create a new task for validating each link, though I think that can also be done ad hoc / as needed. I'm up for going through each link too, but it would take considerably longer.